bundle/1.0.0-1/hive-operator.v1.0.0-1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,620 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
We should watch out for v1alpha1, might be ok now but there should be a more stable version by now, I would be surprised if not.

Reply:
ok